apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedModal/WebsiteEmbedTab.test.tsx

```
@@ -0,0 +1,190 @@
+vi.mock("@/modules/ui/components/button", () => ({
+  Button: (props: {
+    title?: string;
+    "aria-label"?: string;
+    onClick?: () => void;
+    children: React.ReactNode;
+  }) => (
+    <button
+      title={props.title}
+      aria-label={props["aria-label"]}
+      onClick={props.onClick}
+      data-testid="copy-button">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)